src/main/java/frc/robot/Constants.java

```diff
@@ -19,7 +19,9 @@ public final class Constants {
 
 
   public static class OperatorConstants {
-    public static final int kDriverControllerPort = 0;
+    public static final int K_OPERATOR_CONTROLLER_PORT = 0;
```

Comment:
don't use the wpilib `kWhatever` naming constant

additionally this could be shortened to `OP_CONTROLLER` or `OP_PORT`